.gitlab-ci.yml

```diff
@@ -38,3 +38,4 @@ staging:
         K8S_SECRET_MAIL_MAILGUN_API: "https://api.eu.mailgun.net/v3"
         K8S_SECRET_MAILER_EMAIL_BACKEND: "anymail.backends.mailgun.EmailBackend"
         K8S_SECRET_DEFAULT_FROM_EMAIL: "no-reply@hel.ninja"
+        K8S_SECRET_FIELD_ENCRYPTION_KEYS: "f164ec6bd6fbc4aef5647abc15199da0f9badcc1d2127bde2087ae0d794a9a0b"
```

Comment:
👀 should it be here?

Reply:
Probably not. But at the moment it's there just because encrypting something (ssn) without having encryption keys present on runtime will fail (library throws index out of bounds) and before we had a discussion about not having any default values hardcoded in the settings since it's likely that no one remembers to change those for prod. It's not related to the PR but issue was detected during the build of this PR.

Reply:
ok, well, ideally, I guess that it should use something like [this variable](https://github.com/City-of-Helsinki/open-city-profile/blob/develop/.gitlab-ci.yml#L26), but that could be done later on too